lib/ferrum/screencaster.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Ferrum
+  class Screencaster
+      def initialize(page)
+        @page = page
+      end
+
+      def add_frame(params)
+        warn 'frame'
+        @page.command("Page.screencastFrameAck", sessionId: params["sessionId"])
+
+        ts = (Time.now.to_f * 1000).to_i
+        File.binwrite("img_#{ts}.jpeg", Base64.decode64(params["data"]))
```

Comment:
Could maybe put this in a background thread to stop from blocking and get more frames per second.